docker/README.md

```diff
@@ -0,0 +1,61 @@
+# ION using Docker 
+
+This directory contains the automated Docker deployment of ION.  This will create 5 docker containers that contain the core components to run ION.
+
+- bitcoin-core - This container will run the bitcoin-core client.  `NOTE: This is currenlty using testnet as the source chain`
+
+- bitcoin-ion - This is the bitcoin sidetree node for ION.
+
+- ipfs - This is the local IPFS node used by the sidetree.
+
+- mongo - This is the local MongoDB used by the sidetree.
+
+- core-ion - This is the core sidetree node.
```

Comment:
Nit: our convention have been:

ion-bitcoin
ion-ipfs
ion-core

Reply:
Naming can be changed of course, sorry if these names were not the write scheme.